### ISSUE TITLE
Gathering quests don't work

### ISSUE THREAD

**Issue by rretarin** [kryotek777/Sacred-Gold-LobbyServer#2]
Hi, I came across 2 quests that are impossible to complete if you play online. The first is collecting goblin hearts, the counter does not increase. The second is collecting iron bars from orcs, the bars drop like a normal item and do not affect the completion of the quest. If you load onto a local server, then everything works.

**Comment by kryotek777** [comment]
> Hi, I came across 2 quests that are impossible to complete if you play online. The first is collecting goblin hearts, the counter does not increase. The second is collecting iron bars from orcs, the bars drop like a normal item and do not affect the completion of the quest. If you load onto a local server, then everything works.

Hello and thanks for the report! I'll try to reproduce the issue in a bit.

It seems weird that the lobby has anything to do with quests not completing, since when you join a game the client totally disconnects from the lobby and talks directly to the server.

If you can share more about your setup maybe we can figure this out and see if someone can reproduce it

- What version of Sacred are you playing? (Steam, GOG...)
- What language are you playing in? (English, Russian...)
- Are you using any client side mods? (PureHD, Reborn, NL...)
- Are you self hosting the game server or playing on one hosted by someone else? 
- Is the game server vanilla or modded?
- Are you self hosting the lobby or playing on one hosted by someone else? (I'm assuming you're self hosting)
- What's version of the lobby are you using? Did you compile it yourself or is it a precompiled release?
- What platform(s) are you running on? (Windows, Linux, MacOS)

Again, thank you for your time and I hope it will be of help

**Comment by rretarin** [comment]
Sorry, after doing a lot of research to create steps to reproduce the error, I realized that the problem is not in the lobby server. It's a bug in pureHD.

This problem occurs if you use auto-pickup. So, when I first picked up an item with auto-pickup, I saw the error, and then when testing on a local server, I picked up the item with the mouse, thinking that there was no difference.

And in the process of research, I remembered that auto-pickup is not allowed in the network game and this turns on pureHD, but not very well.
